Sum_Of_Cubes/Sum_Of_Cubes.cpp

```diff
@@ -0,0 +1,31 @@
+/*
+    The sum of cubes of first N natural numbers is given
+    by the equation (N * (N + 1) / 2) ^ 2 i.e.
+    1^3 + 2^3 + 3^3 + 4^3 + 5^3.........N^3 sums upto
+    (N * (N + 1) / 2) ^ 2
+*/
+
+#include <bits/stdc++.h>
+using namespace std;
+
+int main()
+{
+    int N;
+    cout << "Enter the value of N: " << endl;
```

Comment:
Please remove spaces around ``<<`` and ``>>`` in cout and cin statements as these are not required and the compiler simply ignores unnecessary spaces.